libs/platform/src/lib/components/form/form-group/form-field/form-field.component.html

```diff
@@ -1,5 +1,5 @@
 <ng-template #renderer>
-    <div [horizontal]="labelLayout === 'horizontal'" fd-form-item class="fd-row">
+    <div [horizontal]="labelLayout === 'horizontal'" fd-form-item class="fd-row fd-row--top">
```

Comment:
I don't think this is the right place to add this modifier. It doesn't affect the alignment. 
![image](https://user-images.githubusercontent.com/53509521/124421627-7c3eaf00-dd7f-11eb-8e13-b279f0260850.png)

The right place might be [here](https://github.com/SAP/fundamental-ngx/blob/main/libs/platform/src/lib/components/form/form-group/form-group.component.html#L17).

@sKudum please correct me if i am wrong.

Reply:
Yes..thats right.

Reply:
@kavya-b you were right. Thanks! I put that class on the wrong component.